### PR TITLE
Diagram layout improvements

### PR DESCRIPTION
**Pull request by a-sr**
Increase ELK version to latest release and activates new layout features.

### CODE REVIEW

**Comment by a-sr** [comment]
All planned changes are implemented. PR is ready for review.

**Comment by lhstrh** [comment]
Looks like we're still stuck with a Maven configuration that doesn't work...

File: org.lflang.diagram/src/org/lflang/diagram/synthesis/util/LayoutPostProcessing.java [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+    /** Reactions and reactors are ordered by their model order if no additional crossing are created. */
+    public static final String TIE_BREAKER = "Optimize Crossings";
+    /**
+     * No crossing minimization is done at all. This requires that actions and timers are sorted based on their model
+     * order.
+     */
+    public static final String FULL_CONTROL = "Full Control";
+    
+    
+    public static final SynthesisOption MODEL_ORDER =
```
Comment (lhstrh) [inline org.lflang.diagram/src/org/lflang/diagram/synthesis/util/LayoutPostProcessing.java:76]: Many methods in this file have no Javadoc. Could you please add those?

File: org.lflang.diagram/src/org/lflang/diagram/synthesis/util/LayoutPostProcessing.java [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    }
+
+    public void orderChildren(List<KNode> nodes) {
+        String modelOrderStrategy = (String) getObjectValue(MODEL_ORDER);
+        if (FULL_CONTROL.equals(modelOrderStrategy)) {
+            nodes.sort(TEXTUAL_ORDER);
+        }
+    }
+
+}
```
Comment (lhstrh) [inline org.lflang.diagram/src/org/lflang/diagram/synthesis/util/LayoutPostProcessing.java:312]: Missing new line.

Comment (lhstrh) [inline org.lflang.diagram/src/org/lflang/diagram/synthesis/util/LayoutPostProcessing.java:312]: Missing new line.

**Review by lhstrh — CHANGES_REQUESTED** [review]
This looks excellent and ready to merge in, except for the missing Javadoc...

**Review by lhstrh — APPROVED** [review]
:rocket: :rocket: :rocket:

**Pull request merged**